vars/odsComponentStageCopyImage.groovy

```diff
@@ -0,0 +1,59 @@
+import org.ods.component.CopyImageStage
+import org.ods.component.IContext
+
+import org.ods.services.OpenShiftService
+import org.ods.services.ServiceRegistry
+import org.ods.util.Logger
+import org.ods.util.ILogger
+
+def call(IContext context, Map config = [:]) {
```

Comment:
test missing ... and you can take a lot from the rolloutstage test above .. (here we should find imported images in the .builds)